### ISSUE TITLE
Some superscription 

### ISSUE THREAD

**Issue by DavideDAmico04** [metz-theolab/bensira-xml#2]
Here:

https://github.com/metz-theolab/bensira-xml/blob/2f4e16175e28a20e499b8309bd1590a7a61954b1/tei_files/ms_a.xml#L915C34-L915C34

the edition has an ע written at the top of the  ל.

we could add these notes in the transcription.

the raw file is okץ

Same issue here: 

https://github.com/metz-theolab/bensira-xml/blob/c2c24a64f22276a4c45deb2ea020a1f789ba06d9/tei_files/ms_a.xml#L1039

we miss a superscript "waw" between the "ה" and the "א"

other superscript missing here:

https://github.com/metz-theolab/bensira-xml/blob/c2c24a64f22276a4c45deb2ea020a1f789ba06d9/tei_files/ms_a.xml#L2325

https://github.com/metz-theolab/bensira-xml/blob/2f4e16175e28a20e499b8309bd1590a7a61954b1/tei_files/ms_a.xml#L3827

https://github.com/metz-theolab/bensira-xml/blob/2f4e16175e28a20e499b8309bd1590a7a61954b1/tei_files/ms_a.xml#L3983

https://github.com/metz-theolab/bensira-xml/blob/2f4e16175e28a20e499b8309bd1590a7a61954b1/tei_files/ms_a.xml#L4108

Here, MS A 15:9 is sopralinear and is missing in the tei 
https://github.com/metz-theolab/bensira-xml/blob/2f4e16175e28a20e499b8309bd1590a7a61954b1/tei_files/ms_a.xml#L4315

In these 2 cases, 

https://github.com/metz-theolab/bensira-xml/blob/2f4e16175e28a20e499b8309bd1590a7a61954b1/tei_files/ms_a.xml#L4366C1-L4367C45

https://github.com/metz-theolab/bensira-xml/blob/2f4e16175e28a20e499b8309bd1590a7a61954b1/tei_files/ms_a.xml#L4370

the words בראשית and אדם are splitted in 2 in the trascription becouse of the superlinear addition that are in the raw but not in the tei.

**Comment by SphRbtHyk** [comment]
Ok, so if I understand correctly, all values that are between <supralinear> </supralinear> in the raw files are missing ?

We need to decide how to take them into account: within the word itself ? the line ?

**Comment by DavideDAmico04** [comment]
Yes, the tei is missing some letters/words/phrases superscripted that are there in the raw file instead. I have listed in the comment all the occurrences where this happens in MS A.
Yes, later, we will think together about the best solution to insert them. I think we have to think almost on a case-by-case basis because special situations result from the edition.

**Comment by DavideDAmico04** [comment]
In MS B

here:

https://github.com/metz-theolab/bensira-xml/blob/0fd903c5f8749156b7d941e4c442e1f56ccaae06/tei_files/ms_b.xml#L66

an entire superscripted line is missing:
6a)    [       עבד משכיל חביב כנפש      וג [בר
